src/main/java/model/Extention.java

```diff
@@ -0,0 +1,41 @@
+package model;
+
+import java.util.Arrays;
+import java.util.function.Supplier;
+
+public enum Extention {
+
+    HTML("html", "text/html;charset=utf-8", "html관련 확장자"),
+    CSS("css", "text/css;charset=utf-8", "css관련 확장자"),
+    NONE("", "", "");
+
+    private final String extension;
+    private final String type;
+    private final String description;
+
+    Extention(String extension, String type, String description) {
```

Comment:
```suggestion
    Extention(String extension, String mimeType, String description) {
```
그런데 description을 어디다 쓸 건지 궁금하네요 ㅎㅎ

Reply:
getter를 제거하고 설명을 달아 놓으면 해당 enum이 어떤 것을 나타내는지 조금 더 잘 나타낼 것으로 생각했습니다. 다음 단계를 진행하면서 삭제하겠습니다.